python/lsst/ts/atdome/dome_csc.py

```diff
@@ -211,6 +211,13 @@ async def do_homeAzimuth(self, data):
         put_final_azimuth_commanded_state = False
         if self.evt_azimuthState.data.homing:
             raise salobj.ExpectedError("Already homing")
+        if self.evt_azimuthState.data.homeSwitch and self.evt_azimuthState.data.homed:
+            self.log.warning(
+                "Not homing, because already homed and on the home switch, "
+                "and the low-level controller cannot handle that."
+                "If you want to home again, first move the dome off the home switch."
```

Comment:
maybe add...
"... off the home switch via a command, or if required, the manual push buttons in the dome"
That'll give observers a heads up that if the command fails they have another option.

Reply:
Good suggestion.